doc/catalog-format-spec.rst

```diff
@@ -358,9 +358,9 @@ following entries:
    [origin.'case(distribution)']
    'debian|ubuntu' = "native:make"
 
-* ``archive-hash``: mandatory string for source archives. A "kind:digest" field
-  that specifies a hash kind and its value. The only accepted kind is SHA512 at
-  this time.
+* ``origin-hashes``: mandatory string for source archives, optional for git
```

Comment:
I don't think it should be optional for git.

Reply:
In accordance with the comment in #210. OK.

Reply:
I have made it a warning for now, with the idea of turning it into an error once the current index is completely updated.